eth/v1alpha1/validator.proto

```diff
@@ -294,22 +296,40 @@ enum ValidatorStatus {
 
 message ValidatorStatusResponse {
     // The corresponding validator status.
-    ValidatorStatus status = 1;
+    ValidatorStatus status = 1 [deprecated = true];
 
     // The block number of the Ethereum proof-of-work chain
-	// where the deposit for the validator was included.
-    uint64 eth1_deposit_block_number = 2;
+    // where the deposit for the validator was included.
+    uint64 eth1_deposit_block_number = 2 [deprecated = true];
 
     // The slot in the beacon chain in which the validator's
     // deposit was included in a block.
-    uint64 deposit_inclusion_slot = 3;
+    uint64 deposit_inclusion_slot = 3 [deprecated = true];
 
     // The epoch in the beacon chain in which the validator
     // is determined as active.
-    uint64 activation_epoch = 4;
+    uint64 activation_epoch = 4 [deprecated = true];
 
     // The position in the activation queue of pending validators.
-    uint64 position_in_activation_queue = 5;
+    uint64 position_in_activation_queue = 5 [deprecated = true];
+
+    // A list of corresponding validator statuses.
+    repeated ValidatorStatus statuses = 6;
```

Comment:
Prefer `status` as plural here to be consistent with the message name `ValidatorStatusResponse`, unless we change it to `ValidatorStatusesResponse` which doesn't sound too pleasant

Reply:
We can't use the name `status` again unless we remove the original deprecated `status` field. 

```
eth/v1alpha1/validator.proto:317:30: "status" is already defined in "ethereum.eth.v1alpha1.ValidatorStatusResponse".
```